replication-api/src/main/java/org/codice/ditto/replication/api/data/Query.java

```diff
@@ -0,0 +1,8 @@
+package org.codice.ditto.replication.api.data;
+
+public interface Query {
+
+  public String getTitle();
+
+  public String getCql();
```

Comment:
❗️ javadoc. Should be annotated with `@Nullable` based on the implementation above.